package.json

```diff
@@ -63,9 +64,13 @@
   "husky": {
     "hooks": {
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
-      "pre-commit": "yarn verify-commit && yarn lint-fix && yarn lint-deps"
+      "pre-commit": "yarn verify-commit && lint-staged"
     }
   },
+  "lint-staged": {
+    "*": "prettier --ignore-unknown --write",
```

Comment:
Prettier supports many file types such as yml, md, html, etc. `--ignore-unknown` will cause prettier to ignore file types it does not support instead of throwing an error.